### ISSUE TITLE
Please avoid node dependency to "natives" module

### ISSUE THREAD

**Issue by bpasero** [attilabuti/vscode-mjml#24]
Hi,

I am a developer on VSCode and wanted to reach out to inform you that this extension is either using or having a dependency to a node module "[`natives`](https://www.npmjs.com/package/natives)" that will soon no longer be supported in VSCode once we updated to node.js 8.x (as part of our Electron 2.0 update which is planned for the near term). 

The full graph is:

```
└─┬ webshot@0.18.0
  └─┬ graceful-fs@3.0.11
    └── natives@1.1.2 
```

It would be great if this dependency could be avoided by either not depending on it anymore or updating the dependencies to newer versions so that this module is no longer pulled in. 

As soon as any code in your extension (including `node_modules`) calls `require("natives")` we will throw an exception to prevent this module from getting loaded.

More details in https://github.com/Microsoft/vscode/issues/47569

Thanks!
Ben
VSCode Team

**Comment by attilabuti** [comment]
Hello @bpasero

Thanks for the warning!
The issue is fixed on 1.2.1.

**Comment by bpasero** [comment]
@attilabuti thanks 👍
